GeosCore/RnPbBe_mod.F90

```diff
@@ -135,7 +135,7 @@ SUBROUTINE CHEMRnPbBe( Input_Opt,  State_Chm, State_Diag, &
 !
 ! !DEFINED PARAMETERS
 !
-    ! Ratio of molecular weights of 210Pb/222Rn
+    ! Ratio of molecular weights of Pb210/Rn222
     REAL(fp), PARAMETER :: Pb_Rn_RATIO = 210e+0_fp / 222e+0_fp
```

Comment:
Should we get this information from the species database instead of hardwiring?

Reply:
@yantosca Yes, eventually, but I have not updated the Rn-Pb-Be code at all in this branch to avoid introducing differences. Modifying the Rn-Pb-Be simulation would likely require discussion with the developers and Transport Working Group.